frontend/js/src/user/Listens.tsx

```diff
@@ -240,12 +240,19 @@ export default class Listens extends React.Component<
     const playingNow = newPlayingNow;
     const { APIService } = this.context;
     try {
-      const metadata = await APIService.lookupRecordingMetadata(
+      const response = await APIService.lookupRecordingMetadata(
         playingNow.track_metadata.track_name,
         playingNow.track_metadata.artist_name,
-        false
+        true
       );
-      playingNow.track_metadata.mbid_mapping = metadata as MBIDMapping;
+      if (response) {
+        const { metadata } = response;
+        playingNow.track_metadata.mbid_mapping = response as MBIDMapping;
```

Comment:
This is currently going to result in a messy listen with lots of unexpected keys, considering the response has a lot more information than what goes into `track_metadata.mbid_mapping`.

Your approach in the following lines with picking properties one by one seems like the best approach.

So I'll ask you to compare the output of the api call ([example](https://api.listenbrainz.org/1/metadata/lookup/?recording_name=Magister+Blues&artist_name=Lemurian+Folk+Songs&metadata=true&inc=release)) with the expected keys in the listen `track_metadata.mbid_mapping` from the `MBIDMapping` type [here](https://github.com/metabrainz/listenbrainz-server/blob/662e847fa78c6623a0ed402143c3481fe843444e/frontend/js/src/utils/types.d.ts#L54) and pick each property one by one, instead of attaching the whole response json.

You'll most likely also be able to drop the hacky `as MBIDMapping` that was there as a bonus :)

Reply:
I have individually assigned the properties as you asked for, but I could not find the 'artists' field in the sample output you gave, so I omitted that field.

Reply:
My bad @KasukabeDefenceForce !

Here is an example of the same API call with artist and tag also passed in the `inc=` query parameter:
https://api.listenbrainz.org/1/metadata/lookup/?recording_name=Magister+Blues&artist_name=Lemurian+Folk+Songs&metadata=true&inc=release%20artist%20tag